test/singa/test_tensor_math.cc

```diff
@@ -120,6 +120,32 @@ TEST_F(TensorMath, SignCpp) {
   EXPECT_EQ(1.0f, dptr1[2]);
 }
 
+TEST_F(TensorMath, SoftPlusCpp) {
+  Tensor aa = a.Clone();
+  Tensor cc = aa - 1.0f;
+  const float *dptr = aa.data<float>();
+  EXPECT_NEAR(0.0f, dptr[0], 1e-5);
+  EXPECT_NEAR(1.0f, dptr[1], 1e-5);
+
+  Tensor p = SoftPlus(cc);
+  const float *dptr1 = p.data<float>();
+  EXPECT_EQ(log(2.0f), dptr1[0]);
+  EXPECT_EQ(log(exp(1) + 1.0f), dptr1[1]);
+}
+
+TEST_F(TensorMath, SoftSignCpp) {
+  Tensor aa = a.Clone();
+  Tensor cc = aa - 1.0f;
+  const float *dptr = aa.data<float>();
+  EXPECT_NEAR(0.0f, dptr[0], 1e-5);
+  EXPECT_NEAR(1.0f, dptr[1], 1e-5);
+
+  Tensor p = SoftSign(cc);
+  const float *dptr1 = p.data<float>();
+  EXPECT_EQ(0.0f, dptr1[0]);
+  EXPECT_EQ(1.0f / 3, dptr1[1]);
+}
+
```

Comment:
@Wentong-DST 
Could you add the cuda test, namely SoftSignCuda and SoftPlusCuda after the line 804 (#ifdef USE_CUDA) . After that, could you please run the test once? Thanks!

Reply:
Hi @chrishkchris , will add test cases and rerun build pipelines. Thanks for reviewing!